lib/key_gen.rb

```diff
@@ -0,0 +1,19 @@
+require_relative './modules/create_defaults'
+
+class KeyGen
+  include Generable
+
+  attr_reader :num
+
+  def initialize(num = generate_random_key)
+    @num = num
+  end
```

Comment:
Keygen is initialized with a default value that can be changed by providing input